src/community/share-research-findings/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Share findings about your users
+description: Find out how to share information on how styles, components and patterns are performing
+section: Community
+theme: Ways to get involved
+layout: layout-pane.njk
+order: 2
+---
+
+<style>
+	code {
+		white-space : pre-wrap !important;
+	}
+</style>
+
+We want to hear how styles, components and patterns perform in your service – it helps us improve the Design System. You can do this by taking part in discussions on Github.
+
+Useful findings to share might be things like:
+
+- summaries of user research you’ve done
+- prototypes or ideas that you’ve tested
+- analytics from your service
+- responses to any calls for information we give in the 'Research on this...' section of our pages
+
+Your service might be in the alpha or beta phases, or you might be iterating the service after going live. 
+
+Feedback from any phase helps us iterate the Design System in line with evolving user needs.
+
+## Where to add findings
+
+Go to our [list of discussions on Github](https://github.com/orgs/alphagov/projects/43/views/1). There are 2 tabs, each showing a list of discussion pages about:
+- things already in the Design System
+- things that could be added in the future
+
+Feedback on both is useful to us. You’ll need to [create a GitHub account](https://github.com/join) to add comments. 
+
+Select the thing you want to share findings about. If what you’re looking for isn’t on the list, read about [how to propose a component or pattern](https://design-system.service.gov.uk/community/propose-a-component-or-pattern/). 
+
+Once you’ve selected the right discussion, use the template below to add your findings.
+
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+<p>
+{% set callout %}
+Share information responsibly
+
+[You must get informed consent from your participants](https://www.gov.uk/service-manual/user-research/getting-users-consent-for-research) before you can share findings of any research they’ve participated in.
+
+Do not share any personally identifiable information about your participants or sensitive information about your service. All the information on Github is open to the public.
+{% endset %}
+{{ govukWarningText({
+html: callout,
+iconFallbackText: "Warning"
+}) }}
+</p>
+## Research template
+
+Use this template to give the community useful context about your findings. Copy and paste it into the Github comments box and add your findings.
```

Comment:
We've got 64x "GitHub" and 12x "Github" on the website

Shall we stick to [**GitHub** with the capital H](https://github.com/logos)?

Reply:
```suggestion
Use this template to give the community useful context about your findings. Copy and paste it into the GitHub comments box and add your findings.
```